### PR TITLE
NCDS recorder swap

### PR DESCRIPTION
**Pull request by mnaguit**
from Basalt SN 560 to Basalt SN 1281

### CODE REVIEW

**Comment by mnaguit** [comment]
I'll add @ozym in the reviewers, as I encountered a number of errors in this PR, mainly linked to whether closing the previous entries in `streams.csv` or stick with the same entries and place the 200hz on continuous mode.

File: install/streams.csv [hunk 1]
```diff
@@ -1269,7 +1269,7 @@ NBSS,20,200,true,false,true,2011-06-21T00:00:10Z,9999-01-01T00:00:00Z
 NCBS,20,50,true,false,false,2015-06-02T01:45:10Z,9999-01-01T00:00:00Z
 NCBS,20,200,true,false,true,2002-10-07T03:00:00Z,9999-01-01T00:00:00Z
 NCDS,20,50,true,false,false,2013-07-24T01:44:02Z,9999-01-01T00:00:00Z
-NCDS,20,200,true,false,true,2002-04-23T00:00:00Z,9999-01-01T00:00:00Z
+NCDS,20,200,true,false,false,2002-04-23T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (ozym) [inline install/streams.csv]: I'm not sure we have addressed the 200Hz going from triggered to continuous issue yet.  I'd suggest keeping the old stream until we can make a plan for the switch (if we deem it necessary).

Comment (mnaguit) [reply install/streams.csv]: Thanks @ozym. I placed the 200hz to triggered mode, instead of continuous.

**Comment by mnaguit** [comment]
Hi @ozym. Changes made on this one too. Kindly review. Thanks!

**Pull request merged**